### ISSUE TITLE
Open to PRs?

### ISSUE THREAD

**Issue by grounded042** [ashtom/hkimport#141]
This project hasn't had any activity in ~3 years. Are you open to PRs? I've done a chunk of work to update things to work with the latest iOS. Happy to contribute it back to this repo if it's open to PRs.

**Comment by ashtom** [comment]
Yes, please send PRs. I should have time over the holidays to review and merge.

**Comment by grounded042** [comment]
Great to hear! I've forked and put a bunch of changes into https://github.com/ashtom/hkimport/compare/main...fitnesstotals:hkimport:main.

I will work on breaking that out into smaller PRs over the holidays.

**Comment by grounded042** [comment]
First PR updating basic workout import: https://github.com/ashtom/hkimport/pull/142

I'll have more coming throughout the week. They'll all build on the first PR here since it adds testing support.

**Comment by grounded042** [comment]
Next PR: #143 to handle more advanced workout data.

**Comment by ashtom** [comment]
@grounded042 Thanks so much, just merged both of them and tested in my local setup. Happy New Year!

**Comment by grounded042** [comment]
@ashtom thank you! One more for ya: #144 

Also, two questions:
1. Any interest in updating the minimum deployment iOS version from 12.0 to a newer version of iOS? There aren't any issues targeting 12.0, but it's no longer supported by Apple.
2. Would you be ok if I updated the project to use SwiftUI?
